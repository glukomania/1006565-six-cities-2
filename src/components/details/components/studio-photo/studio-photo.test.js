import renderer from 'react-test-renderer';
import StudioPhoto from './property-photo';

it(`Property photos are displayed correctly`, () => {
  const propertyPhoto = renderer
    .create(<StudioPhoto
      src={`img/apartment-01.jpg`}
    />)
  .toJSON();
  expect(propertyPhoto).toMatchSnapshot();
});
