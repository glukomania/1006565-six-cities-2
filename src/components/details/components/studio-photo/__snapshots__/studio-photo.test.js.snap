// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property photos are displayed correctly 1`] = `
<div
  className="property__image-wrapper"
>
  <img
    alt="Photo studio"
    className="property__image"
    src="img/apartment-01.jpg"
  />
</div>
`;
